changelogs/fragments/cyberarkconjur-removal.yml

```
@@ -0,0 +1,2 @@
+removed_features:
+  - "conjur_variable lookup - has been moved to the ``cyberark.conjur`` collection. A redirection is active, which will be removed in version 3.0.0 (https://github.com/ansible-collections/community.general/pull/570)."
```

```suggestion
  - "conjur_variable lookup - has been moved to the ``cyberark.conjur`` collection. A redirection is active, which will be removed in version 2.0.0 (https://github.com/ansible-collections/community.general/pull/570)."
```
Ooops, forgot one place :D